crypto/x509/v3_utf8.c

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <stdio.h>
+#include "internal/cryptlib.h"
+#include <openssl/asn1.h>
+#include <openssl/conf.h>
+#include <openssl/x509v3.h>
+#include "ext_dat.h"
+
+const X509V3_EXT_METHOD v3_utf8_list[3] = {
+    EXT_UTF8STRING(NID_subjectSignTool),
+    EXT_UTF8STRING(NID_issuerSignTool),
+    EXT_END
```

Comment:
This is an old useless code artefact.
Please discard it.

Reply:
If i remove this how do i link the extentions NID with type?
now it is done by this:
static const X509V3_EXT_METHOD *standard_exts[] = {
 ...
   &v3_utf8_list[0],
    &v3_utf8_list[1],
...
}
I can replce this f.ex but
static const X509V3_EXT_METHOD *standard_exts[] = {
 ...
   EXT_UTF8STRING(NID_subjectSignTool),,
   EXT_UTF8STRING(NID_issuerSignTool),
...
}
But i will look different

Reply:
I was saying that `EXT_END` is useless.
You could reduce the array to the the 2 used items.

Reply:
fixed